### PR TITLE
docs: update branch to main

### PR DESCRIPTION
**Pull request by fedekunze**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -168,7 +168,7 @@ jobs:
           docker_layer_caching: true
       - run: |
           GAIAD_VERSION=''
-          if [ "${CIRCLE_BRANCH}" = "master" ]; then
+          if [ "${CIRCLE_BRANCH}" = "main" ]; then
```
Comment (fedekunze) [inline .circleci/config.yml:171]: not sure if this is correct @alessio

Comment (alessio) [reply .circleci/config.yml:171]: I wouldn't bother too much. CircleCI integration should be replaced by GH actions. @marbar3778 thoughts?

File: docs/validators/validator-faq.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ Delegators are Atom holders who cannot, or do not want to run a validator themse
 
 Because they share revenue with their validators, delegators also share risks. Should a validator misbehave, each of their delegators will be partially slashed in proportion to their delegated stake. This is why delegators should perform due diligence on validators before delegating, as well as spreading their stake over multiple validators.
 
-Delegators play a critical role in the system, as they are responsible for choosing validators. Being a delegator is not a passive role: Delegators should actively monitor the actions of their validators and participate in governance. For more, read the [delegator's faq](https://hub.cosmos.network/master/delegators/delegator-faq.html).
+Delegators play a critical role in the system, as they are responsible for choosing validators. Being a delegator is not a passive role: Delegators should actively monitor the actions of their validators and participate in governance. For more, read the [delegator's faq](https://hub.cosmos.network/=main/delegators/delegator-faq.html).
```
Comment (alessio) [inline docs/validators/validator-faq.md]:
```suggestion
Delegators play a critical role in the system, as they are responsible for choosing validators. Being a delegator is not a passive role: Delegators should actively monitor the actions of their validators and participate in governance. For more, read the [delegator's faq](https://hub.cosmos.network/main/delegators/delegator-faq.html).
```

**Review by alessio — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (fedekunze) [reply .circleci/config.yml:171]: I thought it was already replaced?

**Pull request merged**